test/components/views/dialogs/SpotlightDialog-test.tsx

```diff
@@ -214,200 +206,165 @@ describe("Spotlight Dialog", () => {
         });
         it("with people filter", async () => {
-            const wrapper = mount(
+            render(
                 <SpotlightDialog
                     initialFilter={Filter.People}
                     initialText={testPerson.display_name}
                     onFinished={() => null}
                 />,
             );
-            await act(async () => {
-                await sleep(200);
-            });
-            wrapper.update();
+            // search is debounced
+            jest.advanceTimersByTime(200);
+            await flushPromisesWithFakeTimers();
 
-            let filterChip = wrapper.find("div.mx_SpotlightDialog_filter");
-            expect(filterChip.exists()).toBeTruthy();
-            expect(filterChip.text()).toEqual("People");
+            let filterChip = document.querySelector("div.mx_SpotlightDialog_filter");
+            expect(filterChip).toBeInTheDocument();
+            expect(filterChip!.innerHTML).toContain("People");
 
-            filterChip.find("div.mx_SpotlightDialog_filter--close").simulate("click");
-            await act(async () => {
-                await sleep(1);
-            });
-            wrapper.update();
-
-            filterChip = wrapper.find("div.mx_SpotlightDialog_filter");
-            expect(filterChip.exists()).toBeFalsy();
+            fireEvent.click(filterChip!.querySelector("div.mx_SpotlightDialog_filter--close")!);
+            jest.advanceTimersByTime(1);
+            await flushPromisesWithFakeTimers();
 
-            wrapper.unmount();
+            filterChip = document.querySelector("div.mx_SpotlightDialog_filter");
+            expect(filterChip).not.toBeInTheDocument();
         });
     });
 
     describe("searching for rooms", () => {
-        let wrapper: ReactWrapper;
-        let options: ReactWrapper;
+        let options: NodeListOf<Element>;
 
         beforeAll(async () => {
-            wrapper = mount(<SpotlightDialog initialText="test23" onFinished={() => null} />);
-            await act(async () => {
-                await sleep(200);
-            });
-            wrapper.update();
-
-            const content = wrapper.find("#mx_SpotlightDialog_content");
-            options = content.find("div.mx_SpotlightDialog_option");
-        });
+            render(<SpotlightDialog initialText="test23" onFinished={() => null} />);
+            // search is debounced
+            jest.advanceTimersByTime(200);
+            await flushPromisesWithFakeTimers();
 
-        afterAll(() => {
-            wrapper.unmount();
+            const content = document.querySelector("#mx_SpotlightDialog_content")!;
+            options = content.querySelectorAll("div.mx_SpotlightDialog_option");
         });
 
         it("should find Rooms", () => {
             expect(options.length).toBe(3);
-            expect(options.first().text()).toContain(testRoom.name);
+            expect(options[0]!.innerHTML).toContain(testRoom.name);
```

Comment:
What are the three options that should be there?
What about checking for the three strings instead?